unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/AvatarShape/AvatarShape.cs

```diff
@@ -70,7 +70,7 @@ public class AvatarShape : MonoBehaviour, IHideAvatarAreaHandler, IPoolableObjec
         internal IPlayerName playerName;
         internal IAvatarReporterController avatarReporterController;
 
-        private StringVariable currentPlayerInfoCardId;
+        private BaseVariable<KeyValuePair<string, string>> currentPlayerInfoCardId;
```

Comment:
we should not use `KeyValuePair` for this matters. Replace with: `(string playerId, string source)`

Reply:
good point, less allocation

Reply:
its a DTO purpose only, no need to be part of the domain.. anonymous type allows us to split the properties which suits better for the needs